### ISSUE TITLE
`WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config`

### ISSUE THREAD

**Issue by adelton** [validatedpatterns/multicloud-gitops#298]
When running
```
./pattern.sh make install
```

per https://validatedpatterns.io/patterns/multicloud-gitops/mcg-getting-started/ I get
```
make -f common/Makefile operator-deploy
make[1]: Entering directory '/home/adelton/project/validatedpatterns/multicloud-gitops'
Checking prerequisites:
  Check for 'git helm oc ansible': OK
  Check for python-kubernetes: OK
  Check for kubernetes.core collection: OK
Checking repository:
  https://github.com/validatedpatterns/multicloud-gitops - branch main: Running inside a container: Skipping git ssh checks
Checking cluster:
  cluster-info: OK
  storageclass: OK
# Retry five times because the CRD might not be fully installed yet
for i in {1..5}; do \
	helm template --include-crds --name-template multicloud-gitops common/operator-install/ -f values-global.yaml --set main.git.repoURL="https://github.com/validatedpatterns/multicloud-gitops" --set main.git.revision=main    | oc apply -f- && break || sleep 10; \
done
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /root/.kube/config
[...]
```

Since I'm running this as an unprivileged user with rootless podman, I believe this warning comes from some container that gets used, so something that should be fixed in the container image.

**Comment by mbaldessari** [comment]
In the past we had a bunch of issues (mostly permissions and selinux iirc) with id mapping with rootless containers, so we just let it run as "root" inside the rootless container and so we bound the host user's home to /root inside the container. This has the drawback that the loose permissions on the KUBECONFIG file on the host will be printed out as /root/.kube/config instead of the correct host path. I guess https://github.com/mbaldessari/multicloud-gitops/commit/52a052fecc82e04410d8a68800a3fc364477244a would fix this. Let me test it on a few platforms before merging it.

**Comment by mbaldessari** [comment]
I tested https://github.com/validatedpatterns/common/pull/425 on RHEL8,9 and F38,39. MCG is now updated, closing this one. Thanks for bringing it up!
